source/config/localhost_config.json

```diff
@@ -7,5 +7,12 @@
        "server_cert": "",
        "server_key": "",
        "root_cert": ""
+    },
+    "moniker_stream_processor": {
```

Comment:
Since these are not applicable for most users, I would rather leave this out of default config.json file. Thoughts? @asumit

Reply:
Removed